hooks/post-checkout

```diff
@@ -34,7 +34,12 @@ merge() {
   GIT_COMMITTER_EMAIL="auto-merge@buildkite" \
   GIT_COMMITTER_NAME="github-merged-pr-buildkite-plugin"
   GIT_MERGE_AUTOEDIT="no" \
```

Comment:
This could be removed, as it is an older version of the --no-edit flag. But it is doesn't do any harm either.

Reply:
Ahh cool, will opt to leave it per the "older scripts" comment in the docs.

Reply:
On second thoughts will remove this... Not needed, and this option is confusing enough